CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 2.10.0-alpha5
```

Comment:
do we need this note? I thought the idea is to bring features to the `master`

Reply:
This was specifically to allow me to release a forked SDK alongside the stable. 
But if we can get this change into `master` that is preferred.

Reply:
changed